wizards/Placement/MatchExpression.tsx

```diff
@@ -1,22 +1,27 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
I was wrestling with the linter trying to figure out how to get it to stop complaining about the non-null assertion. It looks like it's ignored in other files?

Reply:
Extract the property to its own constant and use that instead of props.labelValuesMap. Then typescript can track the type so that your null check will work correctly.

```
export function MatchExpression(props: { labelValuesMap?: Record<string, string[]> }) {
    const labelValuesMap = props.labelValuesMap
    return ( ...
```

Reply:
Thanks for the guidance--I tried it, and I get a message that the null check is still unnecessary. So maybe it's not necessary at all? I'll try taking it out...
<img width="946" alt="image" src="https://user-images.githubusercontent.com/19750917/178087810-bb4f09da-c983-41e1-87cd-955b54f09c71.png">

Reply:
Exactly - because you guarded it with the condition for the ternary. It just doesn't work when you are using properties of an object. Has to be a top-level local variable for some reason.